integrationTests/vm/testIndexer.go

```diff
@@ -73,10 +74,15 @@ func CreateTestIndexer(
 	ti.hasher = testHasher
 	ti.t = t
 	ti.saveDoneChan = make(chan struct{})
+	ti.txsLogsProcessor = &mock.TxLogsProcessorStub{}
 
 	return ti
 }
 
+func (ti *testIndexer) SetTxLogProcessor(txsLogsProcessor process.TransactionLogProcessor) {
```

Comment:
add mock comment?

Reply:
also, why do we need a setter function? in almost every case I've seen in this PR, you call this setter right after the constructor. Might move the setting of txsLogProcessor in the constructor and only call this setter when something custom is needed

Reply:
done